Analysis/GeneralTopics/tars-cloud-basic-analysis.md

```diff
@@ -0,0 +1,114 @@
+- PROTECT
+	- To avoid overloading the system because of burst requests or machine fault, tars handles this scenario in the framework. In order to improve system throughput, server uses request queue to process request asynchronously.
+
+## Supported Platforms :
+- Linux
+- Windows
+- MAC
+
+## Supported Languages :
+- C++
+- Java
+- Nodejs
+- PHP
+- Go
+
+## TARS Projects :
+- TARS LAB
+	- JMeter
+	- Benchmark
+	- TarsJavaStart
+	- TarsTools
+- Service Governance
+	- Tseer
+	- Gateway
+- Storage
+	- DCache
+- Infrastructure
+	- K8Stars
+
+### Link for TARS Landscape: https://tarscloud.org/foundation/projects
+
+# Technical Analysis:
+<img src="https://tarscloud.github.io/TarsDocs_en/assets/tars_top_en.png">
+The above image provides a description of the high-level architecture of TARS cloud.
+1. At the top we have the management system which is an web based interface for managing the platform.<br>
+2. The next layer is the registry which takes care of all the microservices running on the platform and along with their metadata.<br>
+3. The node management module help with the process of deployment, configuration, monitoring and scaling a node on the platform.
+4. TARS cloud also provides support for scalable and dynamic object storage.<br>
+
+### SODA and TARS
+SODA is an open source project which provides an unified framework for managing data storage across different types of storage systems,<br>
+TARS cloud includes built-in support for SODA managed systems, The developers can easily integrate their microservices with SODA-managed<br>
+storage systems using the TARS cloud platform.<br>
+Therefore we can say that SODA and TARS can work seamlessly in building and deploying data-intensive applications that can meet the modern demands<br>
+
+
+
+## Other Links:
+| -- | TARS |
+|--|--|
+| **Website** | https://tarscloud.org/ |
+| **TARS Projects** |  https://tarscloud.org/foundation/projects|
+| **License** | [BSD-3-Clause] Read more below. |
+| **TARS Projects** | https://github.com/TarsCloud |
+| **Twitter** | https://twitter.com/TarsCloud |
+
+
+## License:
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+1. Redistributions of source code must retain the above copyright notice, this
+   list of conditions and the following disclaimer.
+
+2. Redistributions in binary form must reproduce the above copyright notice,
+   this list of conditions and the following disclaimer in the documentation
+   and/or other materials provided with the distribution.
+
+3. Neither the name of the copyright holder nor the names of its
+   contributors may be used to endorse or promote products derived from
+   this software without specific prior written permission.
+
+
+## Contributing:
```

Comment:
No relevant for our analysis. Or you can make it explicit to describe - how to contribute to tars. Otherwise, remove the section